BoomBoxEx2.c

```diff
@@ -1 +1,149 @@
+  NOTE_C5, NOTE_A5, NOTE_E5, NOTE_B4, NOTE_B4
+};
+
+//The length of the notes for the Gamecube start up song 
+int GCnoteduration[] = {
+  16, 16, 16, 16, 16, 16, 16, 16, 16, 16, 16, 16, 16, 16, 16, 16, 16, 16, 16, 16, 16, 16, 16, 16,
+  2,
+};
+
+//The notes for Luigi's Mansion 
+int LMmelody[] = {
+  0, NOTE_E4, NOTE_E4, NOTE_E4, NOTE_E4, 0, NOTE_C4, NOTE_E4, NOTE_E4, 0, NOTE_B3, 0, 0, 0,
+  NOTE_D4, NOTE_D4, NOTE_D4, NOTE_D4, 0, NOTE_B3, NOTE_D4, 0, NOTE_C4, 0, NOTE_B4, NOTE_AS4, 0,
+  NOTE_E3, 0, 0, NOTE_E4, NOTE_E4, NOTE_E4, NOTE_G4, NOTE_E4, 0, NOTE_C4, NOTE_E4, NOTE_FS4,
+  NOTE_E4, 0, NOTE_B3, 0, NOTE_E4, NOTE_D4, NOTE_D4, NOTE_D4, NOTE_D4, NOTE_F4, NOTE_D4, 0,
+  NOTE_C4, NOTE_D4, 0, NOTE_E4, NOTE_D4, 0, NOTE_C4, NOTE_B3, 0, NOTE_A3, 0, 
+};
+
+//The length of the notes for Luigi's Mansion 
+int LMnoteduration[] = {
+  8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 8, 2, 8, 8, 8, 8, 8, 8, 8, 8, 16, 8, 4, 4, 4, 8, 8, 8, 8, 8, 8,
+  8, 2, 8, 8, 8, 8, 2, 8, 8, 8, 8, 2, 2, 8, 8, 8, 2, 8, 8, 8, 8, 16, 8, 16, 16, 8, 16, 16, 8, 4,
+};
+
+//The Deflino Plaza notes
+int SMSmelody[] = {
+  NOTE_B3, NOTE_D4, NOTE_B3, NOTE_G3, NOTE_F3, NOTE_F3, NOTE_E3, NOTE_F3, NOTE_FS3, NOTE_G3,
+  NOTE_E3, NOTE_E3, NOTE_E3, NOTE_G3, NOTE_G3, NOTE_F3, NOTE_F3, NOTE_F3, NOTE_F3, NOTE_E3,
+  NOTE_E3, NOTE_G3, NOTE_G3, NOTE_F3, NOTE_F3, NOTE_F3, NOTE_F3, NOTE_E3, NOTE_E3, NOTE_G3,
+  NOTE_G3, NOTE_F3, NOTE_F3, NOTE_F3, NOTE_F3, NOTE_E3, NOTE_E3, NOTE_G3, NOTE_G3, NOTE_F3,
+  NOTE_F3, NOTE_F3, NOTE_F3, 0, NOTE_F3, NOTE_C4, NOTE_C4, NOTE_E3, NOTE_D3, NOTE_B3, NOTE_B3, 0,
+  NOTE_E3, NOTE_C4, NOTE_C4, NOTE_E3, NOTE_D3, NOTE_B3, NOTE_B3, 0, NOTE_E3, NOTE_C4, NOTE_C4,
+  NOTE_C4, NOTE_D4, NOTE_F3, NOTE_F3, 0, NOTE_F3, 0, NOTE_G3, 0, 0, NOTE_F3, NOTE_C4, NOTE_C4,
+  NOTE_E3, NOTE_D3, NOTE_B3, NOTE_B3, 0, NOTE_E3, NOTE_C4, NOTE_C4, NOTE_E3, NOTE_D3, NOTE_B3,
+  NOTE_B3, 0, NOTE_E3, NOTE_C4, NOTE_C4, NOTE_C4, NOTE_D4, NOTE_F3, NOTE_F3, 0, NOTE_F3, 0,
+  NOTE_G3, 0, NOTE_F3, NOTE_C4, NOTE_F3, NOTE_C4, NOTE_C4, NOTE_C4, NOTE_C4, NOTE_C4, 0, NOTE_D4,
+  NOTE_F3, NOTE_D4, NOTE_F3, NOTE_A3, NOTE_F3, NOTE_A3, NOTE_F3, NOTE_A3, NOTE_F3, NOTE_A3, 
+  NOTE_B3, NOTE_D4, NOTE_B3, NOTE_D4, NOTE_B3, NOTE_D4, NOTE_B3, NOTE_D4, NOTE_F3, NOTE_A3,
+  NOTE_F3, NOTE_A3, NOTE_F3, NOTE_A3, NOTE_F3, NOTE_A3
+};
+
+//the duration of the notes for delfino plaza 
+int SMSnoteduration[] = {
+  8, 8, 8, 8, 8, 8, 8, 8, 8, 2, 4, 
+  4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 
+  4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 
+  8, 8, 4, 4, 4, 4, 4, 2, 8, 8, 4, 4, 4, 4, 8, 2, 8, 8, 4, 4, 4, 4, 8, 8, 8, 8, 8, 1, 2, 8, 8, 4, 4, 4, 4, 4, 2, 8, 8, 4, 4, 4, 4, 8, 2, 8, 8, 4, 4, 4, 4, 8, 8, 8, 8, 8, 1, 2, 4, 4, 4, 4, 4, 4, 4, 4, 
+  4, 2, 2, 2, 
+  4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+};
+
+//int's to simplify code. This way it is easier to read over 
+int buttonState8 = digitalRead(8);
+int buttonState9 = digitalRead(9);
+int buttonState10 = digitalRead(10);
```

Comment:
Good idea! 

This doesn't quite work because `digitalRead(8)` reads the state of the button _now_, so buttonState8 only captures the state of the button at the start of the program.

But this is a great idea to give your buttons a more descriptive name so that it's obvious what the button is for in the code. For example:

```
const int playLuigiButton = 8;
const int playDelfinoButton = 9;
const int playGameCubeButton = 10;
```

Then you'd use them like `if (digitalRead(playLuigiButton) == HIGH)` etc...